cmd/mattermost/commands/team.go

```diff
@@ -410,3 +422,37 @@ func renameTeamCmdF(command *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func modifyTeamCmdF(command *cobra.Command, args []string) error {
+	a, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer a.Shutdown()
+
+	team := getTeamFromTeamArg(a, args[0])
```

Comment:
You are accessing to the args[0], so you need to request at least one argument, if not the app can fail here. I would use `ExactArgs(1)` here. See this https://github.com/spf13/cobra#positional-and-custom-arguments